tests/integration/targets/acme_letsencrypt/dns-challenge-missing-acme-domain.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Test role if acme_domain is not set
```

Comment:
I think checking for errors from a role is a bit more complcated and this seems like a good approach:
https://stackoverflow.com/questions/55521078/how-to-deal-with-errors-coming-from-ansible-roles